crates/router/src/core/payments.rs

```diff
@@ -4489,6 +4508,13 @@ where
     .change_context(errors::ApiErrorResponse::InternalServerError)
     .attach_printable("failed eligibility analysis and fallback")?;
 
+    // Fetch and cache default config for success based routing
+    #[cfg(all(feature = "v1", feature = "dynamic_routing"))]
+    fetch_success_based_routing_configs(state, business_profile)
+        .await
+        .map_err(|e| logger::error!(dynamic_routing_metrics_error=?e))
+        .ok();
```

Comment:
why is this call required? You are anyways calling this in push_metrics function